File: template/tests/test_smoke.py
"""Smoke test — trivial, deterministic. Proves the pytest harness runs.

Run with:  pytest  (or `make test`)
Replace with real tests for the project's modules.
"""


def test_arithmetic_is_deterministic() -> None:
    assert 2 * 21 == 42


def test_toolchain_is_wired_up() -> None:
    assert True

File: template/tests/testthat/test-smoke.R
# Smoke test — trivial, deterministic. Proves the test harness runs.
# Run with:  testthat::test_dir("tests/testthat")  (or `make test`)
# Replace with real tests for the functions in R/.

test_that("arithmetic is deterministic", {
  expect_equal(2L * 21L, 42L)
})

test_that("the test toolchain is wired up", {
  expect_true(TRUE)
})
